src/main/java/mate/academy/spring/controller/CinemaHallController.java

```diff
@@ -0,0 +1,42 @@
+package mate.academy.spring.controller;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import mate.academy.spring.model.CinemaHall;
+import mate.academy.spring.model.dto.CinemaHallRequestDto;
+import mate.academy.spring.model.dto.CinemaHallResponseDto;
+import mate.academy.spring.service.CinemaHallService;
+import mate.academy.spring.service.mapper.CinemaHallMapper;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/cinema-halls")
+public class CinemaHallController {
+    private final CinemaHallService cinemaHallService;
+    private final CinemaHallMapper cinemaHallMapper;
+
+    public CinemaHallController(CinemaHallService cinemaHallService,
+                                CinemaHallMapper cinemaHallMapper) {
+        this.cinemaHallService = cinemaHallService;
+        this.cinemaHallMapper = cinemaHallMapper;
+    }
+
+    @GetMapping
+    public List<CinemaHallResponseDto> getAll(@PathVariable Long id) {
```

Comment:
```suggestion
    public List<CinemaHallResponseDto> getAll(@PathVariable Long id) {
```

Why do we need @PathVariable here?